cistem/protocols/protocol_tomo_resample.py

```diff
@@ -139,46 +129,39 @@ def runTomoResample(self, tomoFile: str):
 """
         self.runJob(prog, args % paramDict)
 
-        self.tomoList.append(paramDict['tomoOutName'])
-
     def createOutputStep(self):
-        labelledSet = self._genOutputSetOfTomograms(
-            self.tomoList, 'resampled')
-        self._defineOutputs(**{OUTPUT_TOMO_NAME: labelledSet})
-        self._defineSourceRelation(self.inTomograms.get(), labelledSet)
-
-    def _genOutputSetOfTomograms(self, tomoList, suffix):
-        tomoSet = SetOfTomograms.create(
-            self._getPath(), template='tomograms%s.sqlite', suffix=suffix)
         inTomoSet = self.inTomograms.get()
-        tomoSet.copyInfo(inTomoSet)
+        outTomoSet = self._createSetOfTomograms("resampled")
+        outTomoSet.copyInfo(inTomoSet)
+        outTomoSet.setSamplingRate(self._getOutputSampling())
```

Comment:
Instead of calling "_getOutputSampling()", could we simply read the pixel size from the header of the newly generated file? Although that probably is slower than what we are doing here, which works fine.